docs/UserGuide.adoc

```diff
@@ -183,197 +183,282 @@ Example:
+image::edit-example-3-command-result.png[width="700"]
 
-* `edit 1 p/91234567 e/johndoe@example.com` +
-Edits the phone number and email address of the 1st person to be `91234567` and `johndoe@example.com` respectively.
-* `edit 2 f/ dp/C:/Users/Seedu/Pictures/superman.png s/instagram janedoe123 s/facebook jane.doe` +
-Marks the 2nd person a `favorite` contact, changes the existing display photo to the new file specified by `C:/Users/Seedu/Pictures/superman.png` and edits the social media accounts to be
-a `Facebook` account with the username `jane.doe`,
-and an Instagram account with the username `janedoe123`.
-* `edit 3 n/Betsy Crower dp/ t/` +
-Edits the name of the 3rd person to be `Betsy Crower`, removes existing display photo and clears all existing tags.
 // end::editperson[]
 
+// tag::favoriteunfavoriteperson[]
 ==== Favoriting a person [Since v1.2]: `fav`
 
-Labels the specified person(s) as a favorite contact. +
-Command format: `fav INDEX [ADDITIONAL INDEXES]`
+To add person(s) as a favorite contact, use the `fav` command.
+
+====
+*Command format*: `[red]#fav# [green]#INDEX [ADDITIONAL INDEXES]...#`
+====
 
 ****
 * Adds the person(s) as a favorite contact at the specified `INDEX` or `INDEXES`.
 * The index refers to the index number shown in the most recently displayed list.
-* The index *must be a positive integer* 1, 2, 3, ...
+* The index must be a positive integer 1, 2, 3, ...
 ****
 
-Examples:
+*Examples*:
+
+* `[red]#fav# [green]#5 6#` +
+Adds the 5th and 6th person as favorite contacts in the most recently displayed list.
+
+.Added 5th and 6th persons as favorite contacts
+image::favorite-multiple-command-result.png[width="700"]
 
-* `fav 1 3` +
-Favorites the 1st and 3rd person in the most recently displayed list.
-* `find Betsy` +
-`unfav 1` +
-Favorites the first person in the list of contacts with the name `Betsy`.
+* `[red]#find# [green]#Bernice#` +
+`[red]#fav# [green]#1#` +
+Adds the first person in the list of contacts with the name `Bernice` as a favorite contact.
+
+.Added the first person i.e. Bernice as a favorite contact
+image::favorite-after-find-command-result.png[width="800"]
 
 ==== Unfavoriting a person [Since v1.2]: `unfav`
 
-Removes the the `favorite` label from the specified person(s). +
-Command format: `unfav INDEX [ADDITIONAL INDEXES]`
+To remove person(s) from the list of favorite contacts, use the `unfav` command.
+
+====
+*Command format*: `[red]#unfav# [green]#INDEX [ADDITIONAL INDEXES]...#`
+====
 
 ****
 * Removes the person(s) from the list of favorite contacts at the specified `INDEX` or `INDEXES`.
 * The index refers to the index number shown in the most recently displayed list.
-* The index *must be a positive integer* 1, 2, 3, ...
+* The index must be a positive integer 1, 2, 3, ...
 ****
 
-Examples:
+*Examples*:
+
+* `[red]#unfav# [green]#1 2#` +
+Removes the 1st and 2nd person from favorite contacts in the most recently displayed list.
```

Comment:
Removes the first and second persons in the most recently displayed person list from your favorite contacts.

Reply:
Updated ✅